src/styles/graybox.scss

```diff
@@ -0,0 +1,6 @@
+.os-raise-graybox {
+  border: 1px solid #D9D9D9;
```

Comment:
Minor: Up to you as far as whether you want to employ this, but I thought I'd point out that you can define [a variable](https://sass-lang.com/documentation/variables) with the color value. A couple of potential benefits of doing that:

* You can define a human-readable name (e.g. so it's easier to know what #d9d9d9 is)
* You can re-use that same value later